test.js

```diff
@@ -59,6 +62,6 @@ function run(i) {
 
 
 var i = 0;
-while (++i < 10) {
```

Comment:
there was an intermittent error and when i set it to 100 I could reliably get that error